src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -499,45 +483,43 @@ public void putUsersByUserId(String userId,
   * @param handler
   * @throws Exception
   */
- private void getPG(PostgresClient postgresClient, User user, Handler<AsyncResult<Integer>> handler) {
-   String pgId = user.getPatronGroup();
-   if (pgId == null) {
-     //allow null patron groups so that they can be added after a record is created
-     handler.handle(io.vertx.core.Future.succeededFuture(1));
-   } else {
-     postgresClient.getById(UserGroupAPI.GROUP_TABLE, pgId, check -> {
-       if (check.succeeded()) {
-         if (check.result() == null) {
-           handler.handle(io.vertx.core.Future.succeededFuture(0));
-         } else {
-           handler.handle(io.vertx.core.Future.succeededFuture(1));
-         }
-       } else {
-         Throwable t = check.cause();
-         logger.error(t.getLocalizedMessage(), t);
-         int retCode = -1;
-         if (t.getLocalizedMessage().contains("uuid")) {
-           retCode = 0;
-         }
-         handler.handle(io.vertx.core.Future.succeededFuture(retCode));
-       }
-     });
-   }
- }
+  private void getPG(PostgresClient postgresClient, User user, Handler<AsyncResult<Integer>> handler) {
+    String pgId = user.getPatronGroup();
+    if (pgId == null) {
+      //allow null patron groups so that they can be added after a record is created
+      handler.handle(io.vertx.core.Future.succeededFuture(1));
+    } else {
+      postgresClient.getById(UserGroupAPI.GROUP_TABLE, pgId, check -> {
+        if (check.succeeded()) {
+          if (check.result() == null) {
+            handler.handle(io.vertx.core.Future.succeededFuture(0));
+          } else {
+            handler.handle(io.vertx.core.Future.succeededFuture(1));
+          }
+        } else {
+          Throwable t = check.cause();
+          logger.error(t.getLocalizedMessage(), t);
+          int retCode = -1;
+          if (t.getLocalizedMessage().contains("uuid")) {
+            retCode = 0;
+          }
+          handler.handle(io.vertx.core.Future.succeededFuture(retCode));
+        }
+      });
+    }
+  }
 
   private boolean checkForDuplicateAddressTypes(User user) {
     Map<String, Integer> countMap = new HashMap<>();
-    if (user.getPersonal() != null &&
-      user.getPersonal().getAddresses() != null) {
-      for(Address address : user.getPersonal().getAddresses()) {
+    if (user.getPersonal() != null
+      && user.getPersonal().getAddresses() != null) {
+      for (Address address : user.getPersonal().getAddresses()) {
```

Comment:
Can this if clause been converted into a guard clause to reduce the indention of the following code and increase readability?

    if (user.getPersonal() == null || user.getPersonal().getAddresses() == null) {
      return false;
    }